packages/fxa-settings/src/components/ButtonDownloadRecoveryKeyPDF/VectorImagesForPdf.tsx

```diff
@@ -3,84 +3,113 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { Svg, Path, Rect } from '@react-pdf/renderer';
+import { Svg, Path, Rect, Circle } from '@react-pdf/renderer';
 
-// To include SVG in PDFs with react-pdf/renderer, SVGs must to be added directly in the format explained in docs.
-// This requires modifying the source code of SVGs to match the expected format.
-// See docs: https://react-pdf.org/svg
+/*
+    To include SVG in PDFs with react-pdf/renderer, SVGs must to be added directly in the format explained in docs.
+    This requires modifying the source code of SVGs to match the expected format.
+
+    Note that support for advanced SVG features is limited (gradients and shadows are notably difficult to port over).
```

Comment:
Are they portable at all, or just difficult? Could note that if possible we should use a simpler version of the SVG (as you have done here).

Reply:
There are some limited linear and circular gradients - maybe it's possible to do something with those to simulate the same effects.